src/applications/virtual-agent/components/webchat/WebChat.jsx

```diff
@@ -25,6 +38,17 @@ const WebChat = ({ token, WebChatFramework, apiSession }) => {
   );
   const userUuid = useSelector(state => state.user.profile.accountUuid);
   const isLoggedIn = useSelector(state => state.user.login.currentlyLoggedIn);
+  const { virtualAgentDecisionLetterDownloadTracking } = useSelector(
+    state => {
+      return {
+        virtualAgentDecisionLetterDownloadTracking:
+          state.featureToggles[
```

Comment:
Is there value in making WebChat a connected component? And adding in a 'mapStateToProps' function?

Reply:
This state code is short term for the feature flag